c/src/ml-api-inference-single.c

```diff
@@ -568,6 +570,72 @@ invoke_thread (void *arg)
   return NULL;
 }
 
+/**
+ * @brief Internal function to get the asynchronous invoke.
+ */
+static int
+ml_single_async_cb (GstTensorMemory * data, GstTensorsInfo * info,
+    void *user_data)
+{
+  ml_single_h single = (ml_single_h) user_data;
+  ml_single *single_h;
+  ml_tensors_info_h _info = NULL;
+  ml_tensors_data_h _data = NULL;
+  unsigned int i;
+  int ret = 0;
+
+  ML_SINGLE_GET_VALID_HANDLE_LOCKED (single_h, single, 0);
+
+  if (!single_h->invoke_async_cb) {
+    /* No callback, do nothing. Internal state changing? */
```

Comment:
How about using ```_ml_error_report```?

Reply:
It is difficult to say this is error state.
Normally it should not be null if async-invoke is enabled. Now single-shot only clears async-callback when it is closing.
Actually, I added this to avoid svace issue :)